acsi_eip_driver/src/servo_node.cpp

```diff
@@ -0,0 +1,212 @@
+/**
+ * @file servo_node.cpp
+ * @brief Tolomatic ACSI servo interface using Ethernet/IP - ROS node to publish
+ *data & handle service calls.
+ *
+ * @author Bill McCormick <wmccormick@swri.org>
+ * @date Feb 13, 2019
+ * @version 0.1
+ * @bug Implicit messaging not functional
+ *
+ * @copyright Copyright (c) 2019, Southwest Research Institute
+ *
+ * @par License
+ * Software License Agreement (Apache License)
+ * @par
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * @par
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <ros/ros.h>
+#include <boost/shared_ptr.hpp>
+
+#include <sensor_msgs/JointState.h>
+
+#include "odva_ethernetip/socket/tcp_socket.h"
+#include "odva_ethernetip/socket/udp_socket.h"
+
+#include "servo.h"
+
+using std::cout;
+using std::endl;
+using boost::shared_ptr;
+using eip::socket::TCPSocket;
+using eip::socket::UDPSocket;
+
+using namespace acsi_eip_driver;
+
+int main(int argc, char* argv[])
+{
+  ros::init(argc, argv, "servo");
+
+  ros::NodeHandle nh("~");
+
+  bool debug;
+  nh.param<bool>("debug", debug, true);
+  ROS_INFO_STREAM("debug is: " << debug);
+  while (debug)
+  {
+    sleep(1);
+    nh.getParam("debug", debug);
+  }
```

Comment:
Do you want to include the "wait here until I connect my debugger" capability in the released version of the driver? If so, the parameter should be documented so it's clear how to use it.

Reply:
Will comment out